conseil-common/src/main/scala/tech/cryptonomic/conseil/common/ethereum/Utils.scala

```diff
@@ -0,0 +1,39 @@
+package tech.cryptonomic.conseil.common.ethereum
+
+import scala.util.Try
+
+import scorex.crypto.hash.Keccak256
+
+/**
+  * Utils methods required by the Ethereum integration.
+  */
+object Utils {
+
+  /**
+    * Normalize hex string by removing 0x prefix.
+    */
+  def remove0x(value: String): String =
+    value match {
```

Comment:
I believe `value.stripPrefix("0x")` would work the same way.

Reply:
good to know :) thanks!